app/src/main/java/de/westnordost/streetcomplete/view/GroupedImageSelectAdapter.java

```diff
@@ -109,22 +109,24 @@ private int getGroup(int index)
 
 	private void expandGroup(int index)
 	{
-		Item item = data.get(index);
-		for (int i = 0; i < item.items.length; i++)
-		{
-			data.add(index + i + 1, item.items[i]);
-		}
-		notifyItemRangeInserted(index + 1, item.items.length);
+        if(index >= 0) {
+            Item item = data.get(index);
+            for (int i = 0; i < item.items.length; i++) {
+                data.add(index + i + 1, item.items[i]);
+            }
+            notifyItemRangeInserted(index + 1, item.items.length);
+        }
 	}
 
 	private void retractGroup(int index)
 	{
-		Item item = data.get(index);
-		for (int i = 0; i < item.items.length; i++)
-		{
-			data.remove(index + 1);
+		if(index >= 0) {
+			Item item = data.get(index);
+			for (int i = 0; i < item.items.length; i++) {
+				data.remove(index + 1);
+			}
+			notifyItemRangeRemoved(index + 1, item.items.length);
 		}
-		notifyItemRangeRemoved(index + 1, item.items.length);
 	}
 
```

Comment:
Not sure what this does and why index should be negative, but this becomes obsolete anyway when you use ImageSelectAdapter.